planner/cascades/implementation_rules.go

```diff
@@ -354,8 +354,9 @@ func (r *ImplTopN) OnImplement(expr *memo.GroupExpr, reqProp *property.PhysicalP
 	switch expr.Group.EngineType {
 	case memo.EngineTiDB:
 		return impl.NewTiDBTopNImpl(topN), nil
+	case memo.EngineTiKV:
+		return impl.NewTiDBTopNImpl(topN), nil
```

Comment:
Could you implement the `TiKVTopNImpl` for this? But it is also ok to let me fix it in the next PR.

Reply:
I will implement the `TiKVTopNImpl` in this PR.